sdk/search/azure-search-documents/azure/search/documents/_internal/aio/_search_index_document_batching_client_async.py

```diff
@@ -33,49 +34,40 @@ class SearchIndexDocumentBatchingClient(HeadersMixin):
     :type index_name: str
     :param credential: A credential to authorize search client requests
     :type credential: ~azure.core.credentials.AzureKeyCredential
+    :keyword bool auto_flush: if the auto flush mode is on. Default to True.
     :keyword int window: how many seconds if there is no changes that triggers auto flush.
-        if window is less or equal than 0, it will disable auto flush
-    :keyword int batch_size: batch size. Default to 1000. It only takes affect when auto_flush is on
-    :keyword persistence: persistence hook. If it is set, the batch client will dump actions queue when it changes
-    :paramtype persistence: PersistenceBase
+        Default to 60 seconds
+    :keyword hook: hook. If it is set, the client will call corresponding methods when status changes
+    :paramtype hook: IndexingHook
     :keyword str api_version: The Search API version to use for requests.
     """
     # pylint: disable=too-many-instance-attributes
-    _ODATA_ACCEPT = "application/json;odata.metadata=none"  # type: str
-    _DEFAULT_WINDOW = 0
-    _DEFAULT_BATCH_SIZE = 1000
 
     def __init__(self, endpoint, index_name, credential, **kwargs):
         # type: (str, str, AzureKeyCredential, **Any) -> None
-
-        api_version = kwargs.pop('api_version', None)
-        validate_api_version(api_version)
-        self._batch_size = kwargs.pop('batch_size', self._DEFAULT_BATCH_SIZE)
-        self._window = kwargs.pop('window', self._DEFAULT_WINDOW)
-        self._auto_flush = self._window > 0
+        super(SearchIndexDocumentBatchingClient, self).__init__(
+            endpoint=endpoint,
+            index_name=index_name,
+            credential=credential,
+            **kwargs)
         self._index_documents_batch = IndexDocumentsBatch()
-        self._endpoint = endpoint  # type: str
-        self._index_name = index_name  # type: str
-        self._index_key = None
-        self._credential = credential  # type: AzureKeyCredential
         self._client = SearchIndexClient(
             endpoint=endpoint, index_name=index_name, sdk_moniker=SDK_MONIKER, **kwargs
         )  # type: SearchIndexClient
         self._reset_timer()
-        self._persistence = kwargs.pop('persistence', None)
 
-    async def _cleanup(self, flush=True, raise_error=False):
+    async def _cleanup(self, flush=True):
         # type: () -> None
         """Clean up the client.
 
         :param bool flush: flush the actions queue before shutdown the client
             Default to True.
-        :param bool raise_error: raise error if there are failures during flushing
-            Default to False which re-queue the failed tasks and retry on next flush.
-        :raises: ~azure.core.exceptions.HttpResponseError
+        :raises: ~azure.core.exceptions.HttpResponseError,
```

Comment:
just a question: does this render correctly, since there's no comma or 'or' between the last two errors?